zerver/lib/send_email.py

```diff
@@ -31,6 +31,7 @@ def build_email(template_prefix, to_user_id=None, to_email=None, from_name=None,
         'realm_name_in_notifications': False,
         'support_email': FromAddress.SUPPORT,
         'verbose_support_offers': settings.VERBOSE_SUPPORT_OFFERS,
+        'server_uri': settings.SERVER_URI,
```

Comment:
I thought we wanted `common_context` from `zerver/context_processors.py` here?  And also, we probably want to use `realm_uri`, not `server_uri`, right?